app/javascript/dashboard/components/widgets/conversation/helpers/emailHeadHelper.js

```diff
@@ -1,7 +1,7 @@
-import emailValidator from 'vuelidate/lib/validators/email';
+import { email as emailValidator } from '@vuelidate/validators';
 
 export const validEmailsByComma = value => {
   if (!value.length) return true;
   const emails = value.replace(/\s+/g, '').split(',');
-  return emails.every(email => emailValidator(email));
+  return emails.every(email => emailValidator.$validator(email));
```

Comment:
👀

Reply:
[Default export](https://github.com/vuelidate/vuelidate/blob/next/packages/validators/src/index.js) from the package exports email like [this](https://github.com/vuelidate/vuelidate/blob/next/packages/validators/src/withMessages/email.js) with a $validator object. 

To use it without that, we need to import it differently. See [docs](https://vuelidate-next.netlify.app/validators.html#using-builtin-validators)